connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
Hi @auhlig, I have not yet changed the code according to resteasy-3.x  As there is client creation process has been in latest 3.x  it will require the corresponding client creation code to be changed.

_when we are planning to release 3.0.5 version for OS4j ? 
 _I will try to push that change before that so this pull request can go along.__